api/pkg/server/organization_handlers.go

```diff
@@ -65,7 +108,7 @@ func (apiServer *HelixAPIServer) getOrganization(rw http.ResponseWriter, r *http
 
 // createOrganization godoc
 // @Summary Create a new organization
-// @Description Create a new organization
+// @Description Create a new organization. Only admin users can create organizations.
```

Comment:
I think we want to make this an environment variable toggle because we do not want this behavior in the SaaS but we do want it for a customer